unity-sdk/AndroidPlugin/OptimoveUnityPlugin/src/main/java/com/optimove/unity/plugin/OptimoveInitProvider.java

```diff
@@ -11,12 +11,19 @@
 
 import com.optimove.android.Optimove;
 import com.optimove.android.OptimoveConfig;
+import com.optimove.android.optimobile.OptimoveInApp;
 
 public class OptimoveInitProvider extends ContentProvider {
     @Override
     public boolean onCreate() {
         Application app = (Application) this.getContext().getApplicationContext();
-        Optimove.initialize(app, new OptimoveConfig.Builder("YOUR OPTIMOVE CREDENTIALS", "YOUR OPTIMOVE MOBILE CREDENTIALS").build());
+        OptimoveConfig.Builder configBuilder = new OptimoveConfig.Builder("YOUR_OPTIMOVE_CREDENTIALS", "YOUR_OPTIMOVE_MOBILE_CREDENTIALS");
+        configBuilder.enableInAppMessaging(OptimoveConfig.InAppConsentStrategy.AUTO_ENROLL);
+        Optimove.initialize(app, configBuilder.build());
+        Optimove.getInstance().pushRegister();
+        Optimove.getInstance().setPushActionHandler(new PushReceiver.PushActionHandler());
+        OptimoveInApp.getInstance().setDeepLinkHandler(new UnityProxy.DeepLinkHandler());
```

Comment:
in the name, can we make obvious that it's an InAppDeepLinkHandler that we are setting, ie rename UnityProxy.DeepLinkHandler --> UnityProxy.InAppDeepLinkHandler

Reply:
done :)